integration/lifecycle/lifecycle_test.go

```diff
@@ -44,6 +44,20 @@ var _ = Describe("Creating a container", func() {
 		})
 	})
 
+	Describe("Docker image download in a container", func() {
+		It("returns a helpful error message when image not found from default registry (#89007566)", func() {
```

Comment:
we don't normally include story IDs in the test code (they're normally already in the commit message), might be best to remove these for consistency with the other tests